src/main/java/org/jitsi/jigasi/transcription/VoskTranscriptionService.java

```diff
@@ -0,0 +1,349 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2020 Atlassian Pty Ltd
```

Comment:
Can you fix this line saying 8x8, you will find it in other files in the repo.

Reply:
Fixed